src/components/PeopleTabel/PeopleTable.tsx

```diff
@@ -0,0 +1,32 @@
+import { PersonRow } from './PersonRow';
+import { Person } from '../../types/Person';
+
+type Props = {
+  people: Person[];
+};
+
+export const PeopleTable: React.FC<Props> = ({ people }) => {
+  return (
+    <table
+      data-cy="peopleTable"
+      className="table is-striped is-hoverable is-narrow is-fullwidth"
+    >
+      <thead>
+        <tr>
+          <th>Name</th>
+          <th>Sex</th>
+          <th>Born</th>
+          <th>Died</th>
+          <th>Mother</th>
+          <th>Father</th>
+        </tr>
+      </thead>
+
+      <tbody>
+        {people.map(person => (
+          <PersonRow person={person} key={person.slug} />
```

Comment:
Ensure that each `Person` object has a unique `slug` property, as it is being used as the key for the `PersonRow` component. This is crucial for React's reconciliation process to work correctly.